console/my-project/how-to/assets/manage-organization-members.mdx

```diff
@@ -1,113 +0,0 @@
----
```

Comment:
were the contents of this page deleted?

Reply:
it was a duplicate one, original mdx is `content/console/my-project/how-to/manage-organization-members.mdx`